src/Cake.Common/Tools/DNVM/Use/DNVMSettings.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Cake.Core.Tooling;
+
+namespace Cake.Common.Tools.DNVM.Use
+{
+    /// <summary>
+    /// Settings for the dnvm command
+    /// </summary>
+    public class DNVMSettings : ToolSettings
+    {
+        /// <summary>
+        /// Gets or sets the runtime to be used
+        /// </summary>
+        public string Runtime { get; set; }
+
+        /// <summary>
+        /// Gets or sets the arch to be used
+        /// </summary>
+        public string Arch { get; set; }

```

Comment:
I agree for the enum but we can't use the bitness of the computer running as default value. For example coreclr runtime doesn't support x86 arch.

We can use an enum for the runtime too (only mono, clr and coreclr values).